pages/common/gh-auth.md

```diff
@@ -0,0 +1,24 @@
+# gh-auth
+
+> Authenticate with GitHub in GitHub cli.
+> More information: <https://cli.github.com/manual/gh_auth_login>.
```

Comment:
```suggestion
> More information: <https://cli.github.com/manual/gh_auth>.
```

Reply:
`gh auth`... is for `gh auth`... both for login and other functions... Why link to specific `_login` site?

Reply:
You wrote `gh_auth_login`, I'm asking to remove the trailing `_login`.

Reply:
:facepalm: Sorry, I'm retarded... and also tired...